.dassie/db/seeds.rb

```diff
@@ -6,7 +6,7 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-Hyrax::Engine.load_seed
+Hyrax::Engine.load_seed if ENV["LOAD_HYRAX_ENGINE_SEED"].present?
```

Comment:
👍🏻

Reply:
oh, maybe this should be `unless ActiveModel::Type::Boolean.new.cast(ENV['SKIP_HYRAX_ENGINE_SEED'])`?

requiring the variable for the preferred default behavior is probably too much. otherwise we need to add this to the docker-compose's environment variables, right?

Reply:
Sorry, I just saw this and pushed a commit with the `if` in place still. I
admittedly wasn't aware this would have implications for a docker-compose
environment. So I can flip the env var value and use `unless`. I'll do that now.